distro_test.go

```diff
@@ -163,6 +175,59 @@ configuration: error in GetConfiguration: failed to convert %q to UTF16
 	}
 }
 
+// The subtests can be parallel but the main body cannot, since it registers a
+// distro, possibly interfeering with other tests.
+//
+//nolint:tparallel
```

Comment:
Ah, this error is due to the fact that the subtests are marked as parallel tests but not the main one, correct?